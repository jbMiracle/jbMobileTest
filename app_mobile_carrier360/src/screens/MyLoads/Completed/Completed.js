import React, { Component } from 'react';
import { View } from 'react-native';
import { Text } from 'native-base';

class Completed extends Component {
  render() {
    return (
      <View testID='completed' style={ { flex: 1, justifyContent: 'center', alignItems: 'center' } }>
        <Text>Completed</Text>
      </View>
    );
  }
}

export default Completed;
