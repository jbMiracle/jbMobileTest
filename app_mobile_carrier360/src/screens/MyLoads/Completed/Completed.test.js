import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Completed from '../Completed/Completed';

it('can render an empty component', () => {
  const renderer = new ShallowRenderer();

  renderer.render(<Completed />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
