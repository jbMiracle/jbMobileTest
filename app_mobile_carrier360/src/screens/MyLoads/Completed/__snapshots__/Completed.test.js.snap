// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render an empty component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="completed"
>
  <Styled(Text)
    virtual={undefined}
  >
    Completed
  </Styled(Text)>
</View>
`;
